### DISCUSSION TITLE
Live packet capture

### DISCUSSION THREAD

**Discussion by Ammarusatx** [Security-Onion-Solutions/securityonion#4415] [Unsupported Versions]
1-Hello everyone: how can I get a live packet capture instead of using the "sudo so-test "command 
2- also the SO is frezed when I'm trying to update or install using Yum 
can you help with this please

**Comment by weslambert** [comment]
Regarding the freezing, could you share any error messages and/or the specs of the box where you installed Security Onion?

You can obtain an ad-hoc PCAP with a tool like tcpdump:

`tcpdump -i <interface> -w <file>`

If you are wanting to capture live traffic in a persistent fashion, setup should do that for you, assuming your TAP/SPAN port are set up correctly.
